### ISSUE TITLE
NosTale wings are not rendered properly

### ISSUE THREAD

**Issue by PSzczepanski1996** [doitsujin/dxvk#1811]
The wings in game NosTale, are rendered in front of character, and should be rendered in the character back. Also animation is bit... so fast? Wonky?

### Software information
Same specs as in: https://github.com/doitsujin/dxvk/issues/1810
Screenshot from Linux:
![wings_debug](https://user-images.githubusercontent.com/21007545/98809129-614cb280-241d-11eb-86b7-107d4b439284.png)
Screenshot from Windows:
https://cdn.discordapp.com/attachments/434701912751996928/776050415367225394/20201111-0-Jarvey.jpg
(Gonna upload better screenshot from Windows soon, I must reboot my PC because of triplebooting).

### System information
- GPU: rx5700 XT
- Driver: AMDGPU (kernel 5.9.6, tested on many other kernel versions before too)
- Wine version: 5.13
- DXVK version: 1.7.2

### Apitrace file(s)
https://share.froggi.es/cedc91762a8e [Password: h4ppycr0w] [8 weeks expiration from today (11 november)]

### Log files
- d3d9.log: 
[NostaleClientX_d3d9.log](https://github.com/doitsujin/dxvk/files/5523732/NostaleClientX_d3d9.log)

**Comment by HoshiYamazaki** [comment]
Another windows screenshot:
![wings_windows](https://user-images.githubusercontent.com/21007545/98811701-8d6a3280-2421-11eb-9b9a-89cc3a48f765.jpg)

Now I see the animation of wings on DXVK is too fast, like 3-5 times faster than on native Windows.

**Comment by K0bin** [comment]
What exactly is wrong about the wings? Looks absolutely identical to me in your screenshots.

**Comment by SveSop** [comment]
![Nos_Tale_Comparison](https://user-images.githubusercontent.com/33990277/98915225-eb525500-24c9-11eb-9317-1ed47d7a88c5.png)
I think i understand what he is talking about.

**Comment by K0bin** [comment]
So it's a depth sorting issue?

**Comment by DadSchoorse** [comment]
@CME42 says this is a depth bias issue on nvidia

**Comment by HoshiYamazaki** [comment]
> @CME42 says this is a depth bias issue on nvidia

It also occurs on my rx5700 XT card, while using AMDGPU.
You can still encounter it on newest proton (7.0-1 by time of writing this comment).

**Comment by Blisto91** [comment]
I tried reproducing this on my R9 380 and mesa git, but i've been unable to. Is it only certain wings?

**Comment by CME42** [comment]
There are two issues here, one is depth bias on nvidia and one is some fixed function fun, i have an nsight trace locally here where it's reproducible.

**Comment by Blisto91** [comment]
But the OP has a AMD GPU. 
Maybe i misunderstand what Nvidia specific issue is

**Comment by CME42** [comment]
Yeah i just got myself confused back then, there is also a depth bias problem on nvidia, but on AMD it's only some fixed function stuff.

**Comment by Blisto91** [comment]
Roger. At least i didn't notice the wings Infront issue when i tested with 1.7.2. and mesa git.
But I'll try to find the correct wing type just to make sure.

**Comment by CME42** [comment]
https://imgsli.com/MTA1MTE2/0/2 dxvk vs wined3d vs dxvk-nv
nsight: https://drive.google.com/file/d/1YlJxu5pcCYZ0HTLsuMqeUazXpW7oYj0A/view?usp=sharing

**Comment by K0bin** [comment]
Fixed by #3501

**Comment by K0bin** [comment]
Apparently it's not fixed.
